test/test_data_providers.py

```diff
@@ -210,8 +210,8 @@ def test_yahoo(self):
                 [8.44268222e-05, 1.00000000e+00]
             ])
             covariance = np.array(
-                [[6.933e+00, -1.643e-04],
-                 [-1.643e-04, 1.199e-06]])
+                [[7.174e+0, -1.671e-04],
```

Comment:
Odd that this changed. Could some dependence, or the platform and/or supporting libraries have changed in some way to alter the outcome. Or is is just down to the new numpy version for which the deprecation fix is being done.